Makefile

```diff
@@ -163,7 +163,7 @@ install-cli: build-cli-cross-platform ## Build and install the Lacework CLI bina
 ifeq (x86_64, $(shell uname -m))
 	mv bin/$(PACKAGENAME)-$(shell uname -s | tr '[:upper:]' '[:lower:]')-amd64 /usr/local/bin/$(CLINAME)
 else
-	mv bin/$(PACKAGENAME)-$(shell uname -s | tr '[:upper:]' '[:lower:]')-386 /usr/local/bin/$(CLINAME)
+	mv bin/$(PACKAGENAME)-$(shell uname -s | tr '[:upper:]' '[:lower:]')-$(shell uname -m) /usr/local/bin/$(CLINAME)
```

Comment:
Do you have an M1 machine? I am assuming that this is fixing the architecture for those machines but just want to confirm.

Reply:
Yep, M1.  `make install-cli` wasn't working, with this change it will move lacework-..-arm64.

Reply:
Of course I'm assuming that `uname -m` will return 386 on that arch.

Reply:
FWIW I am not on M1 this is the output
```
 services git:(main) ✗ uname -m
x86_64
```

Reply:
I'll get rid of this change.  Not worth chasing this atm.